add doc for identityStoreId (#4114)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/apps/index.md

> 

Reviewer: ```suggestion
| identityStoreId       | Specify a valid identity store application ID (from the same org) if this application supports an identity store.                    | String  | TRUE    | NULL   |           |
```

> **Note:** You can't currently manage app provisioning settings via the API. Use the administrator UI.

> ###### Identity Store Id

Reviewer: ```suggestion
###### Identity Store ID
```

Author: done in https://github.com/okta/okta-developer-docs/pull/4114/commits/e00c00ad00dbc75617d9ec9517a747d2a555dd74

> An additional `identityStoreId` can be passed within the `settings` object if application supports identity store. `identityStoreId` should be a valid identity store application id in the same Org.

Reviewer: ```suggestion
You can set an additional `identityStoreId` string parameter within the `settings` object if your application supports an identity store. The `identityStoreId` value must be a valid identity store application `id`. This identity store application must be created in the same org as your application.
```

Author: done in https://github.com/okta/okta-developer-docs/pull/4114/commits/e00c00ad00dbc75617d9ec9517a747d2a555dd74

> 

Reviewer: Suggest to remove this section and put the `identityStoreId` parameter in the **App Settings** parameter table.

Question about the `identityStoreId` value: is it an `appId` of another app created in the same org representing the external identity store app? Or is it a GUID from an external source?

Author: it's appId of another app created in the same org.
